system-tests/lib/system-tests.ts

```diff
@@ -296,12 +296,13 @@ Bluebird.config({
   longStackTraces: true,
 })
 
+let cacheNodeModulesDir
```

Comment:
Why were these changes needed?

Reply:
Good catch. These were needed at one point to standardize output, but I refactored a few things and missed these. They are now removed:

[`fd175c8` (#21599)](https://github.com/cypress-io/cypress/pull/21599/commits/fd175c863f92b811319ae4792a1ee940cc8a3e2f)